src/asmcnc/apps/start_up_sequence/screens/screen_release_notes.py

```diff
@@ -123,46 +125,57 @@ def filter_version_to_filename(character):
     except:
         return False
 
+
 class ScrollReleaseNotes(ScrollView):
-    text = StringProperty('')
+    text = StringProperty("")
+    color_dict = DictProperty(
+        {
+            "background": "e5e5e5ff",
+            "link": "1976d2ff",
+            "paragraph": "333333ff",
+            "title": "333333ff",
+            "bullet": "333333ff",
+        }
+    )
 
-    color_dict = DictProperty({
-                    'background': 'e5e5e5ff',
-                    'link': '1976d2ff',
-                    'paragraph': '333333ff',
-                    'title': '333333ff',
-                    'bullet': '333333ff'})
 
 class ReleaseNotesScreen(Screen):
-
     data_consent_app = None
     user_has_confirmed = False
 
     def __init__(self, **kwargs):
         super(ReleaseNotesScreen, self).__init__(**kwargs)
-        self.start_seq=kwargs['start_sequence']
-        self.sm = kwargs['screen_manager']
-        self.version = kwargs['version']
-        self.l=kwargs['localization']
-
-
+        self.start_seq = kwargs["start_sequence"]
+        self.sm = kwargs["screen_manager"]
+        self.version = kwargs["version"]
+        self.l = kwargs["localization"]
         # Filename consists of just the version digits followed by .txt, so can be found by filtering out non integers from version name
         # Two dots before filename mean parent directory, as file is at the top of the filetree, not in src
-        self.release_notes_filename = '../' + (self.version).replace(".","") + '.txt'
+        self.release_notes_filename = "../" + self.version.replace(".", "") + ".txt"
         self.scroll_release_notes.release_notes.source = self.release_notes_filename
-
         self.update_strings()
 
     def update_strings(self):
-
-        self.version_number_label.text = (self.l.get_str("Software updated successfully to version")).replace(self.l.get_str('version'), self.version)
-        self.please_read_label.text = self.l.get_str("These release notes contain critical information about how SmartBench has changed (in English).")
-        self.url_label.text = self.l.get_str("For full release notes, go to:") + \
-        "\n" + \
-        "https://www.yetitool.com\n/SUPPORT\n/KNOWLEDGE-BASE\n/smartbench1-console-\noperations-software-\nupdates-release-notes"
+        self.version_number_label.text = self.l.get_str(
+            "Software updated successfully to version"
+        ).replace(self.l.get_str("version"), self.version)
+        self.please_read_label.text = self.l.get_str(
+            "These release notes contain critical information about how SmartBench has changed (in English)."
+        )
+        self.url_label.text = (
+            self.l.get_str("For full release notes, go to:")
+            + "\n"
+            + """https://www.yetitool.com
+/SUPPORT
+/KNOWLEDGE-BASE
+/smartbench1-console-
+operations-software-
+updates-release-notes"""
```

Comment:
This formatting looks bizarre, does this work??

Reply:
It surprised me too, but it's fully functional. Not sure I think this is the neater option though...